source/dotnet/Library/AdaptiveCards/AdaptiveBaseElement.cs

```diff
@@ -0,0 +1,92 @@
+using System;
```

Comment:
Is there a copyright header that we need to include on OSS code?

Reply:
We don't currently use a copyright header, though I think we should, according to those [guidelines](https://docs.opensource.microsoft.com/releasing/copyright-headers.html) I found a while back. I've filed #2626 to cover this work.

---
In reply to: [272373665](https://github.com/Microsoft/AdaptiveCards/pull/2560#discussion_r272373665) [](ancestors = 272373665)